CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.6.0 (2019-12-06)
+
+- Removed the `multiselectRowId` input property from the list view grid component. [#55](https://github.com/blackbaud/skyux-list-builder-view-grids/pull/55)
```

Comment:
Doesn't really matter, but don't we usually just say "input" or "property," not both?